components/http/okHttp/gradle/dependencies.gradle

```diff
@@ -13,7 +13,7 @@ dependencies {
     implementation 'io.opentelemetry:opentelemetry-context:1.24.0'
     implementation 'io.opentelemetry:opentelemetry-semconv:1.24.0-alpha'
     implementation 'com.google.guava:guava:31.1-jre'
-    api 'com.squareup.okhttp3:okhttp:4.10.0'
+    implementation 'com.squareup.okhttp3:okhttp:4.10.0'
```

Comment:
revert this change, we're exposing this type in the API surface of the request adapter